ccd-definition/CaseEventToFields/DefendantResponse-prod.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Whats the difference between prod and non-prod?

Reply:
prod is mandatory display, non-prod is complex